testing/tests/DevExpress.ui.widgets.dataGrid/filterRow.tests.js

```diff
@@ -2625,9 +2626,9 @@ QUnit.module('Filter Row with real dataController and columnsController', {
                 assert.strictEqual(loadSpy.getCall(0).args[0].skip, undefined);
             }
 
-            const dropDownList1 = $('.dx-list:eq(0)');
+            const dropDownList1 = $('.dx-list').eq(0);
             assert.strictEqual(dropDownList1.find('.dx-item').length, 91); // 90 rows + (All)
-            assert.strictEqual(dropDownList1.find('.dx-item:eq(1)').text(), 'value10');
+            assert.strictEqual(dropDownList1.find('.dx-item').eq(1).text(), 'value10');
             assert.strictEqual(dropDownList1.find('.dx-item:eq(-1)').text(), 'value99');
```

Comment:
Why are not all such selectors refactored?

Reply:
We need to discuss this moment. Here and in some other places we use not components themselves, but this thing:
`this.rowsView.render($testElement);`
so, the `render` function is not called. Thats my I added styles manually, because I thought that the styles should be added only inside the `render` function of real component

Reply:
Exactly in this case it looks like auto-replace. I can revert changes for the 2631 line.
However, I tried to replace only root selectors like this one:
instead of $('div:eq(0)')
I write $('div').eq(0) and so on.
It's because of implementation of my custom JQuery init function, which I do not want to make more complex